src/sonolus/playlist/list.ts

```diff
@@ -50,7 +62,18 @@ export const installPlaylistList = () => {
                     ),
             ),
             options.keywords,
-        )
+        ).map((playlist) => ({
+            ...playlist,
+            levels: hideSpoilers(
+                serverOptions.spoilers,
+                playlist.levels.map((levelName) => {
```

Comment:
`levelName` is misleading, it can be either a level or a level's name, in the case of it being a level the find will always fail.